stable/grafana/templates/pvc.yaml

```diff
@@ -11,6 +11,11 @@ metadata:
     heritage: {{ .Release.Service }}
   {{- with .Values.persistence.annotations  }}
   annotations:
+{{ toYaml . | indent 4 }}
+  {{- end }}
+  {{- with .Values.persistence.finalizers  }}
+  finalizers:
+    - kubernetes.io/pvc-protection
```

Comment:
toYaml instead?

Reply:
Oops. Yes. This got lost when I rebased and squashed.